appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/servers/ServerImpl.java

```diff
@@ -80,26 +79,23 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    @Override
-    public Server description(String description) {
-        setDescription(description);
-        return this;
-    }
-
     @Override
     public ServerVariables getVariables() {
-        return variables;
+        return variables instanceof ServerVariables || variables == null 
+                ? (ServerVariables) variables
+                : new ServerVariablesImpl(variables);
```

Comment:
FYI: `ServerVariables` got "half-removed". It is deprecated and should be removed in the future. In some places it already was replaced with a `Map<String, SeverVariable>` what causes changes here.